pkg/github/github.go

```diff
@@ -26,6 +29,42 @@ type Github struct {
 	Client *github.Client
 }
 
+// ListTeamMembers lists members of a github team
+func (g *Github) ListTeamMembers(team client.GithubTeam) ([]client.GithubTeamMember, error) {
+	ctx := g.Ctx
+	teamID := team.TeamID
+
+	org, _, err := g.Client.Organizations.Get(ctx, team.Organisation)
+	if err != nil {
+		return nil, err
+	}
+
+	users, _, err := g.Client.Teams.ListTeamMembersByID(ctx, *org.ID, teamID, nil)
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	members := []client.GithubTeamMember{}
+
+	for _, u := range users {
+		// Yes we really do need to do a call for every user,
+		user, _, _ := g.Client.Users.Get(ctx, *u.Login)
+
+		email := ""
+		if user.Email != nil {
```

Comment:
I think the github client has a getter for each of the fields, so you can use:`GetEmail`, which will do this for you. At least I have some recollection of this